### DISCUSSION TITLE
Max-width unacceptable on large screens

### DISCUSSION THREAD

**Discussion by unison3** [facelessuser/pymdown-extensions#1513] [Q&A] [answered]
Almost all documentation I have come across that has been generated with this docs generator has an issue on large screens where the `<div class="doc"></div>` element has a max-width of 680px, it seems to be this way by default... 

I personally get around this by manually going into Chrome dev tools and changing max-width on that div to 100% but it's a pain to do this on every page change or refresh...

I am not familiar with SCSS so I can't see how or where I would change the code base so future docs are not generated this way... If anyone with the know how could fix this and let others know that they can now create more readable docs from the changes, it would be most appreciated and very welcomed by the coding community as a whole.

Thanks for anyone's attention in this matter.

**Comment by facelessuser** [comment, marked as answer]
> Almost all documentation I have come across that has been generated with this docs generator has an issue on large screens where the `<div class="doc"></div>` element has a max-width of 680px, it seems to be this way by default...

I'm uncertain as to what you are referring to.

1. This is not a docs generator, put simply extensions for Python Markdown. Yes, the extensions can be used with Python Markdown to generate documents, but that is not what this project is. Maybe you are using this with a document generator such as MkDocs? If so, maybe your issue is with a MkDocs theme? If so, you would need to take your issue upstream.
2. Nowhere do we generate `<div class="doc"></div>` elements with 680px. We do not inject CSS or set such styling to divs in any of our extensions.

> I am not familiar with SCSS so I can't see how or where I would change the code base so future docs are not generated this way... If anyone with the know how could fix this and let others know that they can now create more readable docs from the changes, it would be most appreciated and very welcomed by the coding community as a whole.

Are you complaining about viewing my project's documentation? Or documentation somewhere else? The SCSS in this project is not used for anything but our own project's documentation. The extensions do not use the SCSS and do not generate any HTML elements with our own document's personal SCSS.

Please be very clear and provide examples as I have no idea what is really being asked, and frankly, I think it is possible you are attributing issues to these extensions which we are not responsible for.
